### PR TITLE
Teststeststests

### PR DESCRIPTION
**Pull request by wasade**
Few more tests

### CODE REVIEW

File: amgut/lib/data_access/ag_data_access.py [hunk 1]
```diff
@@ -719,10 +692,23 @@ def get_barcode_results(self, supplied_kit_id):
         return [dict(zip(col_names, row)) for row in results]
 
     def get_login_info(self, ag_login_id):
-        sql = """select  ag_login_id, email, name, address, city, state, zip,
+        """Get kit registration information
+
+        Parameters
+        ----------
+        ag_login_id : uuid
```
Comment (squirrelo) [inline amgut/lib/data_access/ag_data_access.py]: string thatis a valid UUID, not a UUID object, correct?

File: amgut/lib/data_access/ag_data_access.py [hunk 2]
```diff
@@ -731,10 +717,24 @@ def get_login_info(self, ag_login_id):
...
     def get_survey_id(self, ag_login_id, participant_name):
-        """Return the survey ID associated with a participant or None"""
-        sql = """select survey_id
-                 from ag_login_surveys
-                 where ag_login_id=%s and participant_name=%s"""
+        """Return the survey ID associated with a participant or None
+
+        Parameters
+        ----------
+        ag_login_id : uuid
```
Comment (squirrelo) [inline amgut/lib/data_access/ag_data_access.py]: same here

File: amgut/lib/data_access/test/test_ag_data_access.py [hunk 3]
```diff
@@ -283,17 +286,56 @@ def test_check_if_consent_exists(self):
...
+               {'barcode': '000004213', 'participant_name': 'REMOVED'},
+               {'barcode': '000004214', 'participant_name': 'REMOVED'},
+               {'barcode': '000004215', 'participant_name': 'REMOVED'},
+               {'barcode': '000004216', 'participant_name': 'REMOVED'},
+               {'barcode': '000004218', 'participant_name': 'REMOVED'},
+               {'barcode': '000004219', 'participant_name': 'REMOVED'}]
+        self.assertEqual(obs, exp)
+
+    def test_get_barcode_results_non_existant_id(self):
+        with self.assertRaises(RuntimeError):
```
Comment (squirrelo) [inline amgut/lib/data_access/test/test_ag_data_access.py]: RutmieError here too :(

**Comment by squirrelo** [comment]
couple comments

File: amgut/lib/data_access/ag_data_access.py (hunk 1 above)
Comment (wasade) [inline amgut/lib/data_access/ag_data_access.py]: Good catch

**Comment by squirrelo** [comment]
:+1:

**Comment by wasade** [comment]
@mortonjt @ElDeveloper, able to review as well please?

File: amgut/lib/data_access/ag_data_access.py [hunk 4]
```diff
@@ -741,47 +741,20 @@ def get_user_info(self, supplied_kit_id):
...
+        """Get the results associated with the login ID of the kit
+
+        Parameters
+        ----------
+        supplied_kit_id : str
+            The user's supplied kit ID
+
+        Returns
+        -------
+        dict
```
Comment (ElDeveloper) [inline amgut/lib/data_access/ag_data_access.py]: Line 796 seems to indicate this is really returning a list, am I misunderstanding something?

**Comment by ElDeveloper** [comment]
Looks good, just one comment!

**Comment by mortonjt** [comment]
I think looks good

**Comment by wasade** [comment]
bam bam, @ElDeveloper, good catch

**Comment by ElDeveloper** [comment]
Thanks @wasade

**Pull request merged**